pkg/nitro/metahelm/metahelm.go

```
@@ -993,13 +994,16 @@ func (ci ChartInstaller) DeleteReleases(ctx context.Context, k8senv *models.Kube
 	return nil
 }
 
+// cleanUpNamespace deletes an environment's namespace and ClusterRoleBinding, if they exist
 func (ci ChartInstaller) cleanUpNamespace(ctx context.Context, ns, envname string, privileged bool) error {
 	var zero int64
 	// Delete in background so that we can release the lock as soon as possible
 	bg := meta.DeletePropagationBackground
 	ci.log(ctx, "deleting namespace: %v", ns)
 	if err := ci.kc.CoreV1().Namespaces().Delete(ns, &meta.DeleteOptions{GracePeriodSeconds: &zero, PropagationPolicy: &bg}); err != nil {
-		return errors.Wrap(err, "error deleting namespace")
+		if !k8serrors.IsNotFound(err) {
```

Add a comment explaining that we're swallowing up the error on purpose.

---

Good idea, done